dist/stat_data.py

```diff
@@ -72,10 +72,10 @@ class DhandleStat(Stat):
     prefix = 'data-handle'
     def __init__(self, name, desc, flags=''):
         Stat.__init__(self, name, DhandleStat.prefix, desc, flags)
-class HistoryStat(Stat):
-    prefix = 'history'
+class GCStat(Stat):
+    prefix = 'garbage-collection'
```

Comment:
I would suggest that we use something other than `garbage-collection` for these statistics, that term means too many things and we'll inevitably have a collision with some other kind of garbage collection in the future. I'd reference `checkpoint cleanup` somehow?

To be clear, I won't die on this hill, if someone feels strongly or we don't have time, fbm.

Reply:
I agree with Keith - "garbage collection" is very overloaded. Since this is a user facing change I'd rather get it fixed up before merge if possible.

Reply:
Thanks. Changed the stats to `checkpoint-cleanup` as Keith suggested.